### ISSUE TITLE
Dont allow producer to send messages with value=None to 0.8.0 brokers

### ISSUE THREAD

**Issue by wizzat** [dpkp/kafka-python#124]
python-kafka allows the insertion of a null value into a Kafka topic.  Once this happens, the topic is rather permanently dead and no one can read from it.  Details, including reproduction steps, are here: https://gist.github.com/wizzat/8970493

**Comment by vshlapakov** [comment]
Actually the issue is related with Kafka [bug](https://issues.apache.org/jira/browse/KAFKA-897), not with `kafka-python` implementation. It really exists at `0.8` but was fixed for `0.8.1` (`ConsoleConsumer` failed when a null is passed for the value). I've tested it with `0.8.2` and it works fine:
```
./kafka-console-consumer.sh --topic test --zookeeper zk --from-beginning --property print.key=true                
null    null
null    abc
```
 Should we handle such limitation in kafka-python code?
